composer/lib/dependabot/composer/update_checker/version_resolver.rb

```diff
@@ -285,12 +285,12 @@ def handle_composer_errors(error)
             raise MissingExtensions, [missing_extension]
           elsif error.message.include?("cannot require itself") ||
                 error.message.include?('packages.json" file could not be down')
-            raise Dependabot::DependencyFileNotResolvable, error.message
+            raise Dependabot::DependencyFileNotResolvable, sanitized_message
```

Comment:
Don't we sanitize this downstream? https://github.com/dependabot/dependabot-core/blob/main/updater/lib/dependabot/sentry.rb#L6 if that's not working as expected, maybe we should fix it there?